tree/treeplayer/src/TDFNodes.cxx

```diff
@@ -327,8 +327,8 @@ void TLoopManager::CleanUpNodes()
 
    // forget TActions and detach TResultProxies
    fBookedActions.clear();
-   for (auto readiness : fResProxyReadiness) {
-      *readiness.get() = true;
+   for (const auto &readiness : fResProxyReadiness) {
```

Comment:
changes here are ok, but maybe I'd leave out the `const`: it is technically correct, as we do not modify the pointer ref, but we modify the pointee, so it's a bit misleading.